.github/cleanup-rootfs.sh

```diff
@@ -36,7 +38,7 @@ PATHS=(
 
 function cleanup_packages()
 {
-	if [[ ${RMONLY} == 0 ]]; then
+	if [[ ${APT} == 1 ]]; then
```

Comment:
At line 9 above `APT`` is set to `0` so how is this `if` statment ever going to evaluate to `true` and the enclosing additional cleanup logic executed?

Reply:
Yes, apt part is not used in original file, so I did not change it. 

Maybe someone put it as "You can enable it if you need". 

If we need all the space we can get, I can modify the script accordingly, without any conditional statements.